grammars/css.cson

```diff
@@ -20,294 +20,1205 @@
+        'beginCaptures':
+          '1':
+            'name': 'invalid.deprecated.gradient.function.css'
+          '2':
+            'name': 'punctuation.section.function.begin.bracket.round.css'
+        'end': '\\)'
+        'endCaptures':
+          '0':
+            'name': 'punctuation.section.function.end.bracket.round.css'
+        'name': 'meta.function.legacy-webkit.gradient.css'
+        'patterns': [
+          {
+            'begin': '(?i)(?<![\\w-])(from|to|color-stop)(\\()'
+            'beginCaptures':
+              '1':
+                'name': 'invalid.deprecated.function.css'
+              '2':
+                'name': 'punctuation.section.function.begin.bracket.round.css'
+            'end': '\\)'
+            'endCaptures':
+              '0':
+                'name': 'punctuation.section.function.end.bracket.round.css'
+            'patterns': [
+              {
+                'include': '#property-values'
+              }
+            ]
+          }
+          {
+            'include': '#property-values'
+          }
+        ]
+      }
+      # Misc functions
+      {
+        'begin': '''(?xi) (?<![\\w-])
+          ( annotation
+          | attr
+          | blur
+          | brightness
+          | character-variant
+          | contrast
+          | counters?
+          | cross-fade
+          | drop-shadow
+          | element
+          | fit-content
+          | format
+          | grayscale
+          | hue-rotate
+          | image-set
+          | invert
+          | local
+          | minmax
+          | opacity
+          | ornaments
+          | repeat
+          | saturate
+          | sepia
+          | styleset
+          | stylistic
+          | swash
+          | symbols
+          )
+          (\\()
+        '''
+        'beginCaptures':
+          '1':
+            'name': 'support.function.misc.css'
+          '2':
+            'name': 'punctuation.section.function.begin.bracket.round.css'
+        'end': '\\)'
+        'endCaptures':
+          '0':
+            'name': 'punctuation.section.function.end.bracket.round.css'
+        'name': 'meta.function.misc.css'
+        'patterns': [
+          {
+            'include': '#property-values'
+          }
+          {
+            'match': '[^\'"),\\s]+'
+            'name': 'variable.parameter.misc.css'
+          }
+        ]
+      }
+      # Shapes
+      {
+        'begin': '(?i)(?<![\\w-])(circle|ellipse|inset|polygon|rect)(\\()'
+        'beginCaptures':
+          '1':
+            'name': 'support.function.shape.css'
+          '2':
+            'name': 'punctuation.section.function.begin.bracket.round.css'
+        'end': '\\)'
+        'endCaptures':
+          '0':
+            'name': 'punctuation.section.function.end.bracket.round.css'
+        'name': 'meta.function.shape.css'
+        'patterns': [
+          {
+            'match': '(?<=\\s|^|\\*/)(?i:at|round)(?=\\s|$|/\\*)'
```

Comment:
This `?i:` modifier is inconsistent with how you've been doing it everywhere else: a leading `(?i)`.